calico/release-scripts/tests/test_images.py

```diff
@@ -37,6 +37,31 @@
     "v1.0",
 ]
 
+VPP_IMAGES = ["calicovpp/agent", "calicovpp/vpp", "calicovpp/init-eks"]
+VPP_RELEASE = "v0.14.0"  # TODO This needs to be got from somewhere, not hard-coded
+VPP_EXPECTED_ARCHS = ["amd64"]
+VERSIONS_WITHOUT_VPP = [
```

Comment:
I think you can get rid of all this - the test is only valid for the branch that it's on, so we should assume if the test exists that it should pass for that version, right?